jina/clients/base/helper.py

```diff
@@ -1,9 +1,13 @@
+import argparse
 import asyncio
 import random
 from abc import ABC, abstractmethod
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Optional
 
 from aiohttp import WSMsgType
+from opentelemetry import trace
```

Comment:
can we hide these `imports` when they are needed, so that the dependency is not needed if trace is not activated?

Reply:
We cannot hide the `opentelmetry` api module imports. Otherwise we cannot use any typing. Hiding also doesn't make sense to me because the dependency injection happens in the class constructor for most implementations. These imports are just api's and don't execute code unless some methods are executed.

The sdk imports are already hidden in the `instrumentation` module.